mongo/database.go

```diff
@@ -95,14 +95,18 @@ func (db *Database) Name() string {
 	return db.name
 }
 
-// Collection gets a handle for a given collection in the database.
+// Collection gets a handle for a collection with the given name configured with the given CollectionOptions.
 func (db *Database) Collection(name string, opts ...*options.CollectionOptions) *Collection {
 	return newCollection(db, name, opts...)
 }
 
-// Aggregate runs an aggregation framework pipeline.
+// Aggregate performs an aggregate operation (https://docs.mongodb.com/manual/reference/command/aggregate/) against
+// the database. This requires MongoDB version >= 3.6 and driver version >= 1.1.0.
```

Comment:
Note: here, the lower case term "database" refers to the database on the server.  Elsewhere, you use it to mean the Database object.  I suggest being consistent and using lower case for server and upper case for programmatic object.  Ditto for collection.

Reply:
Good catch. Done.